src/main/java/smartthings/ratpack/sqs/DefaultSqsService.java

```diff
@@ -35,7 +37,8 @@ public DefaultSqsService(
     @Override
     public void onStart(StartEvent event) throws Exception {
         if (config.isEnabled()) {
-            this.sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            Optional.ofNullable(config.getEndpoint()).ifPresent(sqs::setEndpoint);
```

Comment:
Another possibility would be to provide a means of overriding all these default implementations.  Otherwise, we have to potentially maintain all the config options AWS's Sqs client may support.

I was thinking we could do something like this in the SqsModule
```
    @Override
    protected void configure() {
        OptionalBinder.newOptionalBinder(binder(), CircuitBreaker.class)
            .setDefault().toProvider(SimpleCircuitBreaker::new);

        OptionalBinder.newOptionalBinder(binder(), SqsService.class)
            .setDefault().to(DefaultSqsService.class).in(Scopes.SINGLETON);

        OptionalBinder.newOptionalBinder(binder(), SqsConsumerService.class)
            .setDefault().to(DefaultSqsConsumerService.class).in(Scopes.SINGLETON);

        OptionalBinder.newOptionalBinder(binder(), SqsConsumerService.class)
            .setDefault().to(DefaultSqsConsumerService.class).in(Scopes.SINGLETON);

        OptionalBinder.newOptionalBinder(binder(), SqsConfigService.class)
            .setDefault().to(DefaultSqsConfigService.class).in(Scopes.SINGLETON);
    }
```

That way anyone could just provide an alternate implementation of SqsConfigService which would allow user to build any SQS client they want to be used.